src/cmd/troubleshoot/dynakube.go

```diff
@@ -40,14 +40,21 @@ func checkDynakube(troubleshootCtx *troubleshootContext) error {
 	for _, test := range tests {
 		if err := test(troubleshootCtx); err != nil {
 			logErrorf(err.Error())
-			return fmt.Errorf("'%s:%s' Dynakube isn't valid", troubleshootCtx.namespaceName, troubleshootCtx.dynakubeName)
+			return fmt.Errorf("'%s:%s' Dynakube isn't valid. %s",
```

Comment:
I'd remove the "not valid" part. The actual reason is that the Dynakube is missing

Reply:
This loop iterates over multiple tests. It doesn't necessarily error at the "does not exist" part